### PR TITLE
145 imagine rendering issues

### LINKED ISSUE #145: Fix Imagine activities so they are reachable from a URL
# Description

Currently, going to one of the Imagine activities directly from a URL does not work as expected. The URLs lead to the home pages for the activities, but trying to advance to the next page (Avatar selection) leads to a blank screen. We need to fix this so that we can see and interact with the entire activity.

- [x] Fix Avatar selection page rendering issue
- [x] Fix Tic Tac Toe page rendering issue

# Potential fix

It seems like some imports were commented out to prevent the lab from being usable. We should try to un-comment these imports to test if the lab is usable.

In the image below, the React library is commented out, so we should try to un-comment it out.

Image

# Links

1. Imagine activity 1: https://all.rit.edu/Imagine1
2. Imagine activity 2: https://all.rit.edu/Imagine2
3. Imagine activity 3: https://all.rit.edu/Imagine3

### PR DESCRIPTION
**Pull request by Stormkiller-01**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [ ] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/imagine/components/Bias.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Bias = (props) => {
     isModalActive,
     setModalActive,
   } = props;
-  const [setTimerDone] = useState(false);
+  const [TimerDone, setTimerDone] = useState(false);
```
Comment (heathermoses) [inline client/src/components/imagine/components/Bias.jsx]: This is a nitpick but TimerDone should probably be timerDone (camel case)

**Review by heathermoses — APPROVED** [review]
Everything looks good to me!

File: client/src/components/imagine/components/Bias.jsx (hunk 1 above)
Comment (Stormkiller-01) [reply client/src/components/imagine/components/Bias.jsx]: Ok, I'll change that

**Review by Mster115 — APPROVED** [review]

**Review by 2nPlusOne — COMMENTED** [review]
Looks great. I didn't see anything that stood out to me. I like how you formatted the ternary statement on line 54 of Bias.jsx. Much more readable now.

**Comment by 2nPlusOne** [comment]
Are the checkboxes up to date?

> * [ ]  No discrepancies across browsers. Eg. chrome vs safari
> * [ ]  Pages pass W3C Validation for HTML (https://validator.w3.org)
> * [ ]  CSS (https://jigsaw.w3.org/css-validator/)
> * [ ]  Google sign-in is operational
> * [ ]  Accessibility functions
> * [ ]  Pages can scale without distorting page
> * [ ]  No dead links
> * [ ]  Navbar is consistent across the site
> * [ ]  Pages are screen-reader accessible
> * [ ]  Contrast meets standards for accessibility
> * [ ]  Pages are keyboard accessible
> * [ ]  Passes WAVE Evaluation Tool chrome extension

**Pull request merged**